### ISSUE TITLE
LINK2001 ERROR(with sppm.obj,sppm.dll etc)

### ISSUE THREAD

**Issue by firebrain7** [gradientpm/gvpm#2]
After I use cmake to build the vs2013 project successfully, then the compile occurs some errors, most  of them was diminished by me ,but there are some errors not solved yet.
Could you give some advices about it?  thanks a lot !

![image](https://user-images.githubusercontent.com/16367644/67554412-ed703700-f741-11e9-8af2-447a24dbe15a.png)

**Comment by firebrain7** [comment]
I found the misatke occurs on cuda version build.
And on the normal version build, I build successfuly.

**Comment by songuke** [comment]
Thanks Mike. Glad to hear that you built it successfully.

2019年10月26日(土) 13:49 Mike <notifications@github.com>:

**Comment by firebrain7** [comment]
Actually I build successfully without the cuda support.
Once i use the config-win64-msvc2013.py with cuda support ,  there are problems above. The builld with cuda support I am still trying.
thanks, Hua.

**Comment by songuke** [comment]
Yes so far we do not use CUDA support as it is only for the image
reconstruction part.

2019年10月26日(土) 17:30 Mike <notifications@github.com>:
